elastic/kibana
# [Cases] Handle lens actions in Serverless

## Summary

fixes https://github.com/elastic/kibana/issues/160126

This PR 
- hides owner selection while adding lens to case from dashboard for observability and securitySolution serverless
- hides add to new case and add to existing case lens action for Elasticsearch serverless

**How to test**

- Run Observability/Securtiy solution serverless project and add lens visualization to case from dashboard 
- Run ES serverless project and check that lens do not have option to add to case
- Classic kibana works as before (check dashboard from securitySolution and generic dashboard as well)

### Checklist

Delete any items that are not applicable to this PR.


### For maintainers
